### PR TITLE
[CORE] Update seed

### LINKED ISSUE #70: Upgrade theme to Scoped Clarity packages
Upgrade the theme to use the new scoped clarity packages:

- `@clr/angular`
- `@clr/icons`
- `@clr/ui`

### PR DESCRIPTION
**Pull request by hippee-lee**
## Note
This is using the `v0.11.0-alpha.1` version of Clarity and will be updated to the major release once that's finalized. 

## Updates
- Switch to scoped clarity packages (v0.11.0)
- Closes #70

Signed-off-by: Matt Hippely <mhippely@vmware.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "protractor": "~5.1.2",
     "ts-node": "^3.0.4",
     "tslint": "^5.3.2",
-    "typescript": "~2.3.3",
+    "typescript": "^2.5.2",
```
Comment (gnomeontherun) [inline package.json]: This is only if you set Angular to ^5.1, otherwise set `~2.4.2`.

File: package.json [hunk 2]
```diff
@@ -12,19 +12,19 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^4.3.0",
```
Comment (gnomeontherun) [inline package.json:15]: We have a `yarn.lock` file, but it is outdated now. We should probably remove it and optionally add the `package-lock.json`.

**Review by gnomeontherun — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hippee-lee) [reply package.json]: I'm pushing a change in a bit to fix it to only minor versions for angular@5.

File: package.json (hunk 2 above)
Comment (hippee-lee) [reply package.json:15]: Will update here as well.

Comment (hippee-lee) [reply package.json:15]: This is fixed and `package-lock.json` added.

**Review by gnomeontherun — APPROVED** [review]
Just update the README.md and looks good.

**Pull request merged**